Frontend/ui-driver/src/Services/Backend.purs

```diff
@@ -721,7 +721,7 @@ makeDriverRCReq :: String -> String -> Maybe String -> Boolean -> Maybe ST.Vehic
 makeDriverRCReq regNo imageId dateOfRegistration multipleRc category airConditioned = DriverRCReq
     {
       "vehicleRegistrationCertNumber" : regNo,
-      "operatingCity" : "BANGALORE",
+      "operatingCity" : DS.toUpper $ getValueToLocalStore DRIVER_LOCATION,
```

Comment:
Is this change verified from backend as well ?
Also it won't break in case of other merchant right ?

Reply:
@Rohit4997 Why was it hardCoded as "BANGALORE"?